crates/services/gas_price_service/src/common/fuel_core_storage_adapter/storage.rs

```diff
@@ -93,19 +93,16 @@ impl TableWithBlueprint for UnrecordedBlocksTable {
     }
 }
 
-pub struct BundleIdTable;
+pub struct RecordedHeights;
 
-/// The sequence number or bundle id of the posted blocks.
-type BundleId = u32;
-
-impl Mappable for BundleIdTable {
+impl Mappable for RecordedHeights {
```

Comment:
perhaps a comment here about using a fixed key vs different keys to store the last recorded block height?

Reply:
I think I prefer this since we get to explicitly call for the height specified in the metadata. In the case that the metadata is behind the DB height, we can still get the right height.